src/pybind/mgr/dashboard/tox.ini

```diff
@@ -23,6 +23,7 @@ deps =
 [base-test]
 deps =
     -rrequirements-test.txt
+    -e../../../python-common
```

Comment:
let's pause for a moment. this is already included in the `requirements.txt`

Reply:
Right. I added "-e" in roder to follow the pip way.